cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +200,92 @@
+# certificates is currently in the EXPERIMENTAL stage.  Once we have more
+# experience with people using it out in the wild, we will gradually move it to
+# the PRODUCTION stage.
+#
+# Configure ACME, which allows the webpkgserver to automatically renew Signed
+# Exchange certificates.
+#
+# ACME is a protocol that allows automatic certificate renewal. Setting the
+# Enable parameter to true will make webpkgserver automatically request a
+# renewed certificate whenever it has determined that the current certificate
+# is expired or about to expire.
+#
+# If you set up more than one instance of webpkgserver, the recommended practice
+# is to have only one instance handle automatic certificate renewals and let
+# other instances reload the fresh certificate from the disk. It allows
+# certificates to be shared on all webpkgserver instances (which may help
+# minimize the monetary cost, subject to Terms of Service set by your
+# Certificate Authority) and prevents webpkgserver from bombarding your
+# Certificate Authority with certificate requests.
+#
+# webpkgserver uses the ACME library from github.com/go-acme/lego.
+# 
+# For the full ACME spec, see:
+#   https://tools.ietf.org/html/rfc8555 
+[SXG.ACME]
+  # Enable webpkgserver to attempt to auto renew certificates using ACME.
+  #Enable = false 
+
+  # The ACME discovery URL. It is specified by the Certificate Authority that
+  # doles out your certificates. As of August 2020, only DigiCert supports
+  # automatic renewals of signed exchange certificate:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
+  # Required when ACME is enabled.
+  DiscoveryURL = '<Your Discovery URL>'
+
+  # The email address registered to the Certificate Authority for your signed
+  # exchange certificates.
+  # Required when ACME is enabled.
+  Email = 'user@example.com'
+
+  # For the remaining configuration items, it is important to understand the
+  # different challenges employed as part of the ACME protocol:
+  # https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#identifier-validation-challenges
+  # https://letsencrypt.org/docs/challenge-types/
+  # https://certbot.eff.org/docs/challenges.html/
+  #
+  # Note that you do not need to set the fields for all of these challenges. It
+  # is typically sufficient to have a setting for just one of the challenges. If
+  # more than one method is configured, the go-acme/lego library will decide
+  # pick one of them for primary use and use other settings as backup. For
+  # wildcard certificates, nevertheless, DNSProvider is the only supported
+  # method of validation, and others cannot be used. See DNSProvider for more
+  # detail.
+
+  # The http server root directory where the ACME http challenge token should
+  # be deposited. Note that if you are using a reverse-proxy server with
+  # multiple instances of webpkgserver instances, you may need to configure it  
```

Comment:
Sorry, I'm trying hard to understand this sentence and reading the linked page, but I'm still failing to understand. A couple of questions include:

* What does "it" refer to, as in "... to configure **it** in order to ..."?
* What does "this" refer to, as in "...to get ***this*** work."?
* Why does the user need to take extra care when they run multiple webpkgserver instances using the HTTPWebRootDir method?

Reply:
Actually, this whole note about multiple webpkgserver is confusing.  Best to remove it and let the reader research what's best for their particular setup.